src/main/java/mpern/sap/commerce/build/tasks/GlobClean.java

```diff
@@ -57,7 +57,7 @@ public Property<String> getGlob() {
         return glob;
     }
 
-    @Input
+    @InputDirectory
```

Comment:
Using `@Input` is a deliberate choice #31

Reply:
This is unfortunately not compatible with Gradle 8. I will then remove the support for 8.